src/WindowWrapper.cpp

```diff
@@ -0,0 +1,354 @@
+#include <Gosu/Gosu.hpp>
+
+namespace Gosu
+{
+  class WindowForWrapper : public Gosu::Window
+  {
+  struct Callbacks
+  {
+    void (*update)(void *data) = nullptr;
+    void (*draw)(void *data) = nullptr;
+    void (*button_down)(void *data, unsigned btn) = nullptr;
+    void (*button_up)(void *data, unsigned btn) = nullptr;
+    void (*drop)(void *data, const char *filename) = nullptr;
+    bool (*needs_redraw)(void* data) = nullptr; // explicitly set to prevent segfault
```

Comment:
This comment seems a bit random because it is not about `needs_redraw`, but all of these callbacks :)